### ISSUE TITLE
Execution order for bulk parsing is wrong

### ISSUE THREAD

**Issue by Ludee** [OpenEnergyPlatform/open-MaStR#242]
Not sure if this has any effect on the data but I observed the order of the bulk parsing is wrong due to missing leading 0.

```
File 'AnlagenEegSolar_1.xml' is parsed.
Data is cleansed.
File 'AnlagenEegSolar_10.xml' is parsed.
...
File 'AnlagenEegSolar_19.xml' is parsed.
Data is cleansed.
File 'AnlagenEegSolar_2.xml' is parsed.
Data is cleansed.
File 'AnlagenEegSolar_20.xml' is parsed.
```

**Comment by FlorianK13** [comment]
This should not have any effect. It just reads the tables one after another. Or do you think that the order matters here?

**Comment by FlorianK13** [comment]
I have added a new function to correctly sort the list. Together with @deniztepe we will write a unit test for this function, then the branch can be merged.

**Comment by FlorianK13** [comment]
This issue is closed with #257
